administrator/language/en-GB/en-GB.mod_loginsupport.ini

```diff
@@ -0,0 +1,12 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+MOD_LOGINSUPPORT="Login Support Information"
+MOD_LOGINSUPPORT_DOCUMENTATION="Joomla! Documentation"
+MOD_LOGINSUPPORT_FORUM="Joomla! Support forum"
+MOD_LOGINSUPPORT_HEADLINE="You can find help here:"
+MOD_LOGINSUPPORT_NEWS="Joomla! News"
+MOD_LOGINSUPPORT_TITLE="Need Support?"
+MOD_LOGINSUPPORT_XML_DESCRIPTION="This module displays some useful links to Joomla support sites on the login screen. Customisation is possible via language overrides or template override"
```

Comment:
personally i wouldnt bother with the text beginning "customisation...." that is true for every module ;)